frontend/src/scenes/billing/billingLogic.tsx

```diff
@@ -145,10 +162,30 @@ export const billingLogic = kea<billingLogicType>([
                 },
 
                 deactivateProduct: async (key: string) => {
-                    const response = await api.get('api/billing-v2/deactivate?products=' + key)
-                    lemonToast.success('Product unsubscribed')
-                    actions.reportProductUnsubscribed(key)
-                    return parseBillingResponse(response)
+                    actions.resetUnsubscribeError()
+                    try {
+                        const response = await api.getResponse('api/billing-v2/deactivate?products=' + key)
+                        const jsonRes = await getJSONOrNull(response)
+                        lemonToast.success('Product unsubscribed')
+                        actions.reportProductUnsubscribed(key)
+                        return parseBillingResponse(jsonRes)
+                    } catch (error: any) {
+                        if (error.detail && error.detail.includes('open invoice')) {
+                            actions.setUnsubscribeError({
+                                detail: error.detail,
+                                link: <Link to={values.billing?.stripe_portal_url}>See Invoices</Link>,
```

Comment:
```suggestion
                                link: <Link to={values.billing?.stripe_portal_url} target="_blank">View invoices</Link>,
```

No period here with the capitalized 'Invoices' makes it look like something is missing. I think this should fix it - might want to pull down and verify - it should show an external link icon, and lowercased "invoices". I also prefer "View" over "see" but that's just me maybe.

Reply:
You're right, I think view invoices is more standard.